taglib/ogg/xiphcomment.cpp

```diff
@@ -40,6 +40,19 @@ namespace
   typedef List<FLAC::Picture *> PictureList;
   typedef PictureList::Iterator PictureIterator;
   typedef PictureList::Iterator PictureConstIterator;
+
+  /* returns the tag value for the given map[key] - however it ensure that map
```

Comment:
NB: `bool Ogg::XiphComment::contains()` will still create entries for non exist elements - to avoid breaking that contract we use `find(...)` to determine existance instead of _fixing_ and using `contains()`